plugins/wallabag.koplugin/main.lua

```diff
@@ -241,6 +242,14 @@ function Wallabag:addToMainMenu(menu_items)
                             self:saveSettings()
                         end,
                     },
+                    {
+                        text = _("Remove downloaded from remote"),
+                        checked_func = function() return self.is_remove_downloaded_from_remote end,
```

Comment:
Please write it on multiple lines, that's easier to deal with in the future (+ better diffs).